gis/libosmium/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           boost 1.0
+
+github.setup        osmcode libosmium 2.17.0 v
+github.tarball_from archive
+revision            0
+
+categories          gis
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+license             Boost-1 public-domain
+
+description         A fast and flexible C++ library for working with OpenStreetMap data
+
+long_description    Low-level: this is designed to be a building block \
+                    for writing a very customized decoder for a stable \
+                    protobuf schema. If your protobuf schema is \
+                    changing frequently or lazy decoding is not \
+                    critical for your application then this approach \
+                    offers no value: just use the C++ API that can be \
+                    generated with the Google Protobufs protoc \
+                    program.
+
+checksums           rmd160  3923cb87ea091aa69a23ca2ee517ea438ac9fe56 \
+                    sha256  4a7672d7caf4da3bc68619912b298462370c423c697871a0be6273c6686e10d6 \
+                    size    513404
+
+installs_libs       no
+
+compiler.cxx_standard 2011
+
+configure.args-append \
+                    -DBUILD_TESTING=OFF \
+                    -DBUILD_EXAMPLES=OFF
+
+depends_build-append port:protozero
```

Comment:
this doesn't follow the whitespace as implied by the modeline (i.e., multiples of four spaces):
```
depends_build-append \
                    port:protozero
```

Reply:
Ok